### ISSUE TITLE
add methodology of Ye et al., 2017​ that uses wavelet transform as feature generated step

### ISSUE THREAD

**Issue by JustinGOSSES** [JustinGOSSES/predictatops#13]
add methodology of Ye et al., 2017​ that uses wavelet transform as feature generated step.

See slide 5 of https://onedrive.live.com/edit.aspx?resid=C028D856AB4461D!2889&ithint=file%2cpptx&authkey=!AAJOHk5oEZJhcuk for example image

**Comment by JustinGOSSES** [comment]
Tagged as hackathon project but it might be too ambitious as hackathon project, really not a good handle on it.

**Comment by bluetyson** [comment]
https://publications.csiro.au/publications/#publication/PIcsiro:EP149165

https://github.com/GeoscienceAustralia/geo-wavelets/tree/373d7f9e24ea08d2c7556e5824002e715430925a/preprocessing

**Comment by JustinGOSSES** [comment]
Can't figure out what this is supposed to be used on https://github.com/GeoscienceAustralia/geo-wavelets

Rasters of seismic data ?

**Comment by bluetyson** [comment]
Yeah, rasters of stuff... haven't tried seismic, but several others
